### PR TITLE
Prevent danmaku block  video's subtitles and more translations

### PR DESCRIPTION
**Pull request by WooLNinesun**
作者您好，最近採用 Dplayer 做為網站的撥放器，想避免擋到字幕和更多一點的翻譯，做了一些改變。
如果寫得不好，還請指教！非常感謝能有這麼棒的開源撥放器。

## src/options:
    - Browser langage support  IE6 ~ IE8,
    - Use lowercase to prevent case typo from user,
## src/i18n.js:
    - Replace "zh" and "us" with language-code to support more language trans,
    - add zh-tw translation.
## src/html.js:
    - Add option.danmaku.margin to prevent danmaku block  video's subtitles,
## demo/index.html:
    - Just demo danmaku.margin,

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -23,7 +23,10 @@ <h2 id="quick-start">Quick Start</h2>
                 },
                 danmaku: {
                     id: '9E2E3368B56CDBB4',
-                    api: 'https://api.prprpr.me/dplayer/'
+                    api: 'https://api.prprpr.me/dplayer/',
+                    margin: {
+                        bottom:'15%'
```
Comment (sinchang) [inline demo/index.html]: : 后加空格

File: src/i18n.js [hunk 2]
```diff
@@ -1,37 +1,70 @@
...
+        'Rolling': '滚动',
+        'Input danmaku, hit Enter': '输入弹幕，回车发送',
+        'About author': '关于作者',
+        'DPlayer feedback': '播放器意见反馈',
+        'About DPlayer': '关于 DPlayer 播放器',
+        'Loop': '洗脑循环',
+        'Speed': '速度',
+        'Opacity for danmaku': '弹幕透明度',
+        'Normal': '正常',
+        'Please input danmaku content!': '要输入弹幕内容啊喂！',
```
Comment (sinchang) [inline src/i18n.js]: 感觉不要加语气词比较好

File: src/option.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ module.exports = (option) => {
         autoplay: false,
         theme: '#b7daff',
         loop: false,
-        lang: navigator.language.indexOf('zh') !== -1 ? 'zh' : 'en',
+        lang: (navigator.language || navigator.browserLanguage).toLowerCase().indexOf('zh-cn') !== -1 ? 'zh-cn' : 'en-us',
```
Comment (DIYgod) [inline src/option.js]: The default `lang` can be more universal, like this
```js
lang: (navigator.language || navigator.browserLanguage).toLowerCase()
```

File: src/i18n.js [hunk 4]
```diff
@@ -1,37 +1,70 @@
...
             return text;
         }
-        else if (this.lang === 'zh') {
-            return tranZH[text];
-        }
+        else if (
+            // add language-code list here
+            this.lang === 'zh-cn' ||
+            this.lang === 'zh-tw'
+        ) { return tranTxt[this.lang][text]; }
```
Comment (DIYgod) [inline src/i18n.js]:
```js
if (tranTxt[this.lang]) {
    return tranTxt[this.lang][text];
}
else {
    return text;
}
```

File: src/i18n.js (hunk 2 above)
Comment (DIYgod) [reply src/i18n.js]: 这个没事..这是我之前写的2333

**Comment by WooLNinesun** [comment]
上述都已經修正了！

**Comment by sinchang** [comment]
👍

**Comment by DIYgod** [comment]
Thanks.
We have a developer group in telegram, do you use telegram and want to join it?

**Comment by WooLNinesun** [comment]
Sure, this is my link [WooLNinesun](https://t.me/WooLNinesun)

**Pull request merged**